docs/5.x/faq.md

```diff
@@ -114,3 +114,8 @@ If the installed kernel exposes the option fs.may_detach_mounts we recommend alw
 sysctl -w fs.may_detach_mounts=1
 echo "fs.may_detach_mounts = 1" >> /etc/sysctl.d/10-may_detach_mounts.conf
 ```
+
+## Cluster Status
+
+Gravity provides the `gravity status` command to view [Cluster Status](/cluster/#cluster-status).
+This tool can help identify issues with the Cluster.
```

Comment:
nit: cluster (non-capitalized)

Reply:
It looks like we are using "Cluster" (capitalized) when referring to a gravity cluster in the docs. Is there a difference when using "Cluster" vs "cluster"?  If not, then its probably better to leave it capitalized for consistency.

Reply:
Um - ok, it does look like this - sorry, non issue.